CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 ## [next-release]
 
+### Added
+- Introduce Jest unit testing framework
```

Comment:
Not sure if I would include this in the changelog. It's not part of the deliverable to the customer.

Reply:
We add internal changes to the changelog proceeded by `Internal: `